app/models/zone.rb

```diff
@@ -48,7 +52,31 @@ def find_master_server
     active_miq_servers.detect(&:is_master?)
   end
 
+  def self.create_maintenance_zone
+    MiqRegion.seed
```

Comment:
@bdunne - when MiqServer.seed calls Zone.seed, because it's dependency then Zone.seed should call MiqRegion.seed too, right? Because we want to be consistent